turbopy/core.py

```diff
@@ -345,7 +353,8 @@ def __init__(self, grid_data: dict):
         self.dr = None
         self.parse_grid_data()
 
-        self.r = self.r_min + (self.r_max - self.r_min) * self.generate_linear()
+        self.r = (self.r_min + (self.r_max - self.r_min) *
```

Comment:
My linter flagged this line as being too long. It's ok to clean up code when adding docs.